packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-11/actions.sh

```diff
@@ -2,4 +2,8 @@
 
 . ./upgrade-test-scripts/env_setup.sh
 
-# Core-eval contract upgrade
+# CWD is agoric-sdk
+here=./packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-11
```

Comment:
The upgrade-test-scripts are copied to the CWD.

```suggestion
here=./upgrade-test-scripts/agoric-upgrade-11
```

Reply:
I originally tried it that way and it failed. The fact that it passes in ci currently is pretty good evidence that it should stay the way it is, no?

Reply:
oh. copied. hm.